deploy/data_exporter/template.yaml

```diff
@@ -0,0 +1,49 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  data_exporter
+
+  Exports data from the prod database, cleans it and puts the resulting dump in an S3 bucket
+
+Globals:
+  Function:
+    Timeout: 600  # 10 minutes
```

Comment:
This probably isn't an issue for YNR, but the max time a lambda can run for is 15 mins
I'd be really interested to know how long it takes to dump a really big DB with lots of GIS polygons in it (e.g: Every Election)

Reply:
As talked about on a call: the time this takes is mostly bound by the Lambda instance size. If we run into this problem we can bump the size to something higher.